### ISSUE TITLE
server.freq not used

### ISSUE THREAD

**Issue by Neightly** [arl/statsviz#41]
```go
func SendFrequency(freq time.Duration) OptionFunc {
	return func(s *server) error {
		s.freq = freq
		return nil
	}
}
```
```go
func sendStats(conn *websocket.Conn) error {
	tick := time.NewTicker(defaultSendFrequency)
	defer tick.Stop()

	var stats stats
	for {
		select {
		case <-tick.C:
			runtime.ReadMemStats(&stats.Mem)
			stats.NumGoroutine = runtime.NumGoroutine()
			if err := conn.WriteJSON(stats); err != nil {
				return err
			}
		}
	}
}
```

sendStats always use defaultSendFrequency, instead of server.freq.
so that SendFrequency does nothing.

**Comment by arl** [comment]
Thank you for reporting.
This has already been resolved in #37 
Which version are you using?
Latest is v0.2.2
https://pkg.go.dev/github.com/arl/statsviz@v0.2.2

**Comment by arl** [comment]
Resolved in #37
